cmake/presets/common.json

```diff
@@ -20,14 +19,16 @@
             "name": "release",
             "hidden": true,
             "cacheVariables": {
-                "CMAKE_BUILD_TYPE": "RelWithDebInfo"
+                "CMAKE_BUILD_TYPE": "RelWithDebInfo",
+                "CMAKE_INSTALL_PREFIX": "/usr/local"
```

Comment:
this should actually be only in `os-linux.json` (which reminds me of, we didn't think about FreeBSD, damn, there it should also be like that)